docs/tutorials/index.md

```diff
@@ -22,12 +49,6 @@ Active learning increases performance in STEM {cite:p}`freeman_active_2014`
 SWC lessons learned {cite:p}`wilson_software_2016`
 
 
-```{image} ../images/tutorials.png
-:alt: Tutorials in action
-:width: 500px
-:align: center
-```
-
```

Comment:
It's too far down to comment on that line (now 62) - the badge text refers to "helpers" instead of "tutorial leads"